samplomatic/builders/specs.py

```diff
@@ -14,17 +14,20 @@
 
 import enum
 from dataclasses import dataclass
+from typing import Literal, TypeAlias
 
 import numpy as np
 
 from ..aliases import CircuitInstruction, Parameter, Qubit, StrRef
-from ..annotations import DressingMode, InjectionSite, VirtualType
+from ..annotations import ChangeBasisMode, DressingMode, InjectionSite, VirtualType
 from ..partition import QubitPartition
 from ..synths import Synth
 
 EMPTY_IDXS = np.empty((0, 0), dtype=np.intp)
 EMPTY_IDXS.setflags(write=False)
 
+FrameChangeMode: TypeAlias = ChangeBasisMode | Literal["local_clifford"]
```

Comment:
i wonder if this would be cleaner/more extensible if we cut ties with the ChangeBasis enum, wdyt?

```suggestion
FrameChangeMode: TypeAlias = Literal["local_clifford", "pauli_prep", "pauli_meas"]
```